packages/opencensus-core/src/index.ts

```diff
@@ -90,3 +89,10 @@ export * from './metrics/metric-registry';
 // GAUGES CLASSES
 export * from './metrics/gauges/derived-gauge';
 export * from './metrics/gauges/gauge';
+
+
+// Stats singleton instance
+import {BaseStats} from './stats/stats';
+import {Stats} from './stats/types';
+const globalStats: Stats = BaseStats.instance;
```

Comment:
Is the `: Stats` type specifier needed here?

Those are useful for enforcing the type assignment to a structural type like `const x: Y = {...}`, but since `BaseStats.instance` already has a type I would think it isn't needed.

Reply:
@draffensperger Are you ok with my comment or you have better approach to handle this?